src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Common.props

```diff
@@ -0,0 +1,15 @@
+﻿<!--
+  WARNING: These targets are intended for building Microsoft's ASP.NET Core repos and are not intended
+  for use outside of Microsoft.
+-->
+<Project>
+  <!-- Set nothing if handled in project file or already done in outer build. -->
+  <PropertyGroup>
+    <EnableApiCheck Condition=" '$(EnableApiCheck)' == '' AND '$(MSBuildRuntimeType)' == 'Core' ">true</EnableApiCheck>
```

Comment:
I wasn't aware of a design discussion about this. Please re-consider. Making VS builds and build.cmd as similar as possible is a highly desirable goal.

If the only concern is that api check errors might impede a developer's inner loop, there are better solutions. Two come to mind: (1) make api check failures a warning in Debug but an error in Release. (2) detect "BuildingInsideVisualStudio" and demote errors to being non-fatal.